core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2718,6 +2718,21 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_FILE_SYSTEM_OPERATION_RETRY_CACHE_ENABLED =
+      new Builder(Name.MASTER_FILE_SYSTEM_OPERATION_RETRY_CACHE_ENABLED)
+          .setDefaultValue(true)
+          .setDescription("If enabled, each fs operation will be tracked on all masters, "
```

Comment:
prefer the full expression for the sake of our documentation: `... each filesystem operation ...`